### PR TITLE
Branch002

### PR DESCRIPTION
**Pull request by Poikka001**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,11 +8,13 @@
     <version>3.1.4</version>
     <relativePath/> <!-- lookup parent from repository -->
   </parent>
+
   <groupId>mate.academy</groupId>
-  <artifactId>appbookstore</artifactId>
+  <artifactId>liquibase</artifactId>
```
Comment (Rommelua) [inline pom.xml:13]: Why?

**Review by Rommelua — CHANGES_REQUESTED** [review]
PR code is not match the task
image

File: src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java [hunk 2]
```diff
@@ -9,6 +11,7 @@
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateRequestBookDto {
+    @NotNull
     private String title;
 
     private String author;
```
Comment (Sanderfuck) [inline src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java:17]: You validate title but not author

**Review by Sanderfuck — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java [hunk 3]
```diff
@@ -9,8 +11,9 @@
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateRequestBookDto {
+    @NotNull
     private String title;
-
+    @NotNull
     private String author;
 
     private String isbn = "" + new Random().
```
Comment (kerrrusha) [inline src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java:19]: Strings should be joined using `Joiner.on(',')` as specified in the code review rules.

File: src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java [hunk 4]
```diff
@@ -9,8 +11,9 @@
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateRequestBookDto {
+    @NotNull
```
Comment (kerrrusha) [inline src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java:14]: The field `title` is not marked as `final`. Consider making it `final` if it is a constant.

File: src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java [hunk 5]
```diff
@@ -19,6 +22,8 @@ public class CreateRequestBookDto {
             mapToObj(Integer::toString).
             collect(Collectors.joining());
 
+    @NotNull
+    @Min(0)
     private BigDecimal price;
```
Comment (kerrrusha) [inline src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java:27]: The field `price` is not marked as `final`. Consider making it `final` if it is a constant.

File: src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java [hunk 6]
```diff
@@ -9,8 +11,9 @@
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateRequestBookDto {
+    @NotNull
     private String title;
-
+    @NotNull
     private String author;
```
Comment (kerrrusha) [inline src/main/java/mate/academy/intro/dto/CreateRequestBookDto.java:17]: The field `author` is not marked as `final`. Consider making it `final` if it is a constant.

File: src/main/java/mate/academy/intro/exeption/CustomGlobalExceptionHandler.java [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+@ControllerAdvice
+public class CustomGlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(
+            MethodArgumentNotValidException ex,
+            HttpHeaders headers,
+            HttpStatusCode status,
+            WebRequest request
+    ) {
+
```
Comment (kerrrusha) [inline src/main/java/mate/academy/intro/exeption/CustomGlobalExceptionHandler.java:27]: redundant blank line

**Review by kerrrusha — APPROVED** [review]

**Pull request merged**